### ISSUE TITLE
Can't farm Torchflower

### ISSUE THREAD

**Issue by WarnDa** [LMBishop/Quests#652]
### Description of Bug

When using CoreProtect hook, it's impossible to make a quest about torchflower.
It's considered as placed by a placed (even if you just place the torchflower_seeds and wait for it to growth)

### Steps to Reproduce

Make a quest like : 
```
tasks:
  torchflower:
    type: "blockbreakcertain"
    check-coreprotect: true
    check-coreprotect-time: 1296000
    amount: 2400
    block: TORCHFLOWER
  torchflower2:
    type: "farmingcertain"
    amount: 2400
    block: TORCHFLOWER
 ```
None of them will work when you will break torchflower.
But if you set "check-coreprotect: false" this work.

### Environment

- Minecraft version: 1.20.4
- Server software: git-Paper-459 (MC: 1.20.4
- Quests version: 3.15-10fc3ae

### Agreements

- [X] I am running the latest version of Quests
- [X] I am certain this issue is unique and a similar issue is not currently open
- [X] I am using a version of Minecraft which is supported by Quests

### Other

Might be the same for "pitcher_plant"

**Comment by NoltoxGit** [comment]
Hello @WarnDa,

Firstly, the task: `blockbreakcertain` is no longer valid since v3.13 of the plugin.

[All information about this task on the wiki](https://quests.leonardobishop.com/task-types/blockbreak-(task-type).html#blockbreak-task-type)

**Comment by Krakenied** [comment]
Its valid, these task have been merged in 3.13 and it can be used as an alias

**Comment by WarnDa** [comment]
Edit : working on playerblocktracker addon, but not coreprotect
